pkg/virt-operator/resource/generate/components/secrets.go

```diff
@@ -367,7 +367,7 @@ func MergeCABundle(currentCert *tls.Certificate, currentBundle []byte, overlapDu
 		certBytes := cert.EncodeCertPEM(crt)
 
 		// don't add the current CA multiple times
-		if reflect.DeepEqual(certBytes, current) {
+		if equality.Semantic.DeepEqual(certBytes, current) {
```

Comment:
This compares bytes so probably `reflect.DeepEqual` is the right thing here. WDYT?

Reply:
yes, you're right